kano_feedback/DataSender.py

```diff
@@ -459,16 +465,22 @@ def send_question_response(question_id, answer, interactive=True):
     payload = {
         'email': get_email(),
         'username': get_mixed_username(),
-        'answers': [
-            {
-                'question_id': question_id,
-                'text': answer,
-                'tags': ['os', 'feedback-widget']
-            }
-        ]
+        'answers': []
     }
 
-    # Send data
+    for q in answers:
+        answer_block = { 'question_id': q.split(':')[0], 'text': q.split(':')[1], 'tags': tags }
+        payload['answers'].append (answer_block)
```

Comment:
All this can almost certainly be done in 1 block with list comprehension:

```python
 payload = {
    'email': get_email(),
    'username': get_mixed_username(),
    'answers': [
        {
            'question_id': q.split(':')[0],
            'text': q.split(':')[1],
            'tags': tags
        } for q in answers
    ]
}
````